### PR TITLE
Dockerfile: reorganized, upgraded to PHP 5.6.6

### PR DESCRIPTION
**Pull request by bryanlatten**
@jimdelois
@ingluisjimenez
@bossjones  -- php 5.6.6 (!)

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -3,6 +3,12 @@ MAINTAINER Bryan Latten <latten@adobe.com>
 
 ENV DEBIAN_FRONTEND noninteractive
 
+# Install pre-reqs, security updates
```
Comment (Attamusc) [inline Dockerfile:6]: Why'd this get removed?

File: Dockerfile [hunk 2]
```diff
@@ -29,28 +35,39 @@ RUN pecl install igbinary-1.2.1 && \
...
-# Enable write functionality for Apache
+# Enable apache rewrite module
 RUN a2enmod rewrite
 
-# Replace apache security file with local one
-COPY ./apache2/conf-available/security.conf /etc/apache2/conf-available/security.conf
-
-ADD ./container/run.sh /run.sh
-ADD ./container/cleanup.sh /cleanup.sh
+# Install New Relic monitoring tools
```
Comment (Attamusc) [inline Dockerfile:41]: Very cool

File: Dockerfile (hunk 1 above)
Comment (bryanlatten) [inline Dockerfile:6]: @Attamusc are you looking at the diffs backwards?

Comment (Attamusc) [inline Dockerfile:6]: looooooooool green means go

**Comment by Attamusc** [comment]
:+1:

**Comment by ingluisjimenez** [comment]
:+1:

**Pull request merged**